packages/charts/src/renderers/accessors/xy-accessors.ts

```diff
@@ -8,11 +11,26 @@ export interface IXYDataAccessors {
     /** Additional custom keys to match the base interface */
     [key: string]: DataAccessor | undefined;
 }
+export interface IXYSeriesAccessors {
+    /** Color of the series */
+    color?: SeriesAccessor;
+    /** Marker for the series */
+    marker?: SeriesAccessor;
+    /** Additional custom keys to match the base interface */
+    [key: string]: SeriesAccessor | undefined;
+}
+export interface IXYAccessors extends IAccessors {
+    data: IXYDataAccessors;
+    /** Series level accessors - e.g. for colors, markers, etc. */
+    series: IXYSeriesAccessors;
+}
```

Comment:
Do we still need these interfaces? It seems like we could just remove the abstract keyword from the original XYAccessors class and that would be all that's needed.